### DISCUSSION TITLE
frontend-vagas web page

### DISCUSSION THREAD

**Discussion by mateusvahl** [frontendbr/forum#1242] [Dicas]
Oi pessoal, tudo bem?
Assim como muitos, estou sempre observando as vagas postadas no  [frontendbr/vagas/issues](https://github.com/frontendbr/vagas/issues).

A poucos dias atrás, queria criar algo em react/redux/styled-components, mas sem desenvolver outro todo app, dai surgiu a ideia de criar uma pagina simples que mostrasse todos as vagas cadastradas no frontendbr.

Queria compartilhar com vocês essa versão 1.0, ideias sugestões são bem vindas, PR's também
Live: https://frontend-vagas.herokuapp.com/

Código fonte: https://github.com/mateuspv/frontend-vagas

Abraços.

**Comment by kvnol** [comment]
Sensacional!

**Comment by laforetmoreno** [comment]
Muito maneiro, parabéns!

**Comment by felipefialho** [comment]
Parabéns pelo trampo @mateuspv !

**Comment by luizfelipehh** [comment]
Ficou show! Parabéns

**Comment by kivervinicius** [comment]
mto bom

**Comment by ViniciusGularte** [comment]
Ficou muito bom, o modal aqui pra mim esta ficando encima do botão de fechar ele,  quando se clica para visualizar detalhes da vaga.
